### PR TITLE
Do not display errors when the player is paused or ended for timeouts.

### PR DESCRIPTION
**Pull request by bclwhitaker**

### CODE REVIEW

File: test/index.html [hunk 1]
```diff
@@ -4,12 +4,13 @@
...
   <title>Video.js Errors</title>
   <link rel="stylesheet" href="../node_modules/qunitjs/qunit/qunit.css">
+  <link rel="stylesheet" href="../src/videojs.errors.css">
 </head>
 <body>
   <div id="qunit"></div>
   <div id="qunit-fixture"></div>
   <script src="../node_modules/qunitjs/qunit/qunit.js"></script>
-  <script src="../node_modules/sinon/lib/sinon/util/fake_timers.js"></script>
+  <script src="../node_modules/sinon/pkg/sinon.js"></script>
```
Comment (gkatsev) [inline test/index.html]: is there no direct file for the fake timers?

**Comment by gkatsev** [comment]
Do we have a test that checks to see that timeout error isn't triggered when the player is paused or ended?

**Comment by bclwhitaker** [comment]
we do.  i updated them so they would have exposed this bug in the PR?

**Comment by gkatsev** [comment]
Oh, didn't realize that's what the updated tests tested :)
LGTM, then.

**Comment by saxena-gaurav** [comment]
LGTM.

**Pull request merged**